src/check.c

```diff
@@ -298,9 +298,9 @@ static int bad_name(DOS_FILE * file)
 	return 0;
 
     for (i = 0; i < MSDOS_NAME; i++) {
-	if (name[i] < ' ' || name[i] == 0x7f)
+	if ((name[i] < ' ' && !(i == 0 && name[0] == 0x05)) || name[i] == 0x7f)
 	    return 1;
-	if (name[i] > 0x7f)
+	if (name[i] > 0x7f || (i == 0 && name[0] == 0x05))
```

Comment:
I think you added this in the revision, but what is the purpose? This will cause it to fail in Atari mode if it starts with 0x05, is that intended?

Reply:
Ou, right, this one is used only for atari mode. Purpose was as in above line translate first 0x05 to 0xE5.

Reply:
Removed.